demo/image_classifier_interface_load/run.py

```diff
@@ -0,0 +1,29 @@
+import gradio as gr
+
+images = ["cheetah1.jpeg", "cheetah1.jpg", "lion.jpg"]
+
+
+img_classifier = gr.Interface.load(
+    "models/google/vit-base-patch16-224", examples=images, cache_examples=True
+)
+
+
+def func(img, text):
```

Comment:
Demo to test using interface as function still works and that cache examples=True works now